src/crimson/osd/object_context_loader.h

```diff
@@ -210,6 +210,22 @@ class ObjectContextLoader {
     }
   };
 
+  /**
+   * create_cached_obc_from_push_data
+   *
+   * Creates a fresh cached obc from passed oi and ssc.
+   * obc must not already in cache.
+   */
+  ObjectContextRef create_cached_obc_from_push_data(
+    const object_info_t &oi,
+    SnapSetContextRef ssc) {
+    auto obc = obc_registry.get_cached_obc(oi.soid).first;
```

Comment:
nit:
```
auto [obc, existed] = obc_registry.get_cached_obc(oi.soid);
// obc must not already in cache.
ceph_assert(!existed)
```

Reply:
It actually can exist, but it can't be valid.  I actually added and then removed that assert.

Reply:
Repushed with a fix to the method comment.